content/en/real_user_monitoring/frustration_signals/_index.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Frustration Signals
+kind: documentation
+further_reading:
+- link: '/real_user_monitoring/dashboards/frustration_signals_dashboard/'
+  tag: 'Documentation'
+  text: 'Frustration Signals Dashboard'
+- link: '/real_user_monitoring/explorer'
+  tag: 'Documentation'
+  text: 'Learn about the RUM Explorer'
+- link: '/real_user_monitoring/session_replay'
+  tag: 'Documentation'
+  text: 'Learn about Session Replay'
+---
+
+## Overview
+
+<div class="alert alert-warning">
+For access to frustration signals, contact <a href="/help">Datadog Support</a>.
+</div>
+
+Frustration signals identify the highest points of user friction in your application by surfacing moments where users exhibit frustration. 
+
+RUM collects three types of frustration signals:
+
+Rage Clicks
+: A user clicks on an element more than three times in a one-second rolling window.
+
+Dead Clicks
+: A user clicks on a static element thinking it is interactive.
+
+Error Clicks
+: A user clicks on an element right before a JavaScript error occurs.
+
+## Requirements
+
+First, you need the Browser RUM SDK version >= 4.9.0.
+
+To start collecting frustration signals, add the following to your SDK configuration:
+
+```
+DD_RUM.init({
+  trackInteractions: true,
+  trackFrustrations: true
+})
+```
+
+Frustration signals require actions. Enabling `trackFrustrations` automatically enables `trackInteractions`.
+
+## Usage
+
+Frustration signals appear as high-level data points representing sources of user frustration on the [**RUM Applications** page][1]. The [RUM Explorer][2] displays a count of frustration signals as a column in the **List** visualization. 
+
+### Application list
+
+Hover over the list of browser sessions and click on a session to observe a user's frustrated click behavior. Or, click **Frustrated Sessions** to access sessions with a frustration signal. 
+
+### Explore the frustration signals dashboard
+
+The **Frustration Signals** dashboard provides an overview of frustration levels across your application, showing you topics like the most frustrated users and pages with the highest amount of frustration signals.
+
```

Comment:
this is where we should add the dashboard, can you use the image i sent over yesterday? that one is what the final will look like

Reply:
I included the image in the RUM Frustration Signals Dashboard page already, I don't think we should duplicate the image in two places.